src/emqx_app.erl

```diff
@@ -37,13 +37,13 @@ start(_Type, _Args) ->
     print_otp_version_warning(),
     print_banner(),
     %% Load application first for ekka_mnesia scanner
-    _ = load_ce_modules(),
+    _ = application:load(emqx_modules),
     ekka:start(),
     {ok, Sup} = emqx_sup:start_link(),
     ok = start_autocluster(),
     ok = emqx_plugins:init(),
     _ = emqx_plugins:load(),
-    _ = start_ce_modules(),
+    _ = application:ensure_all_started(emqx_modules),
```

Comment:
The application for `emqx-modules` is a different implementation in the Enterprise Edition. It is started by loaded_plugins.

So I think this should be handled separately, and the enterprise and open source versions should not share this logic.

Reply:
Thanks. My bad.
I misread `ifndef` as `ifdef` below.

Reply:
fixed.